tools/crs_converter/CRSconverter.xml

```diff
@@ -0,0 +1,316 @@
+<tool id='crs_converter' name='CRSconverter' version='1.0.0+galaxy@VERSION_SUFFIX@' profile='21.05'>
+    <description>to modify Coordinate Reference Systems from a shapefile</description>
+    <macros>
+        <token name='@VERSION_SUFFIX@'>0</token>
+        <import>CRS_macros.xml</import>
+    </macros>
+    <requirements> 
+        <requirement type='package' version='4.3.3'>r-base</requirement> 
+        <requirement type='package' version='1.0-16'>r-sf</requirement>
+        <requirement type='package' version='1.3.1'>r-tidyr</requirement>
+    </requirements>
+    <command detect_errors='exit_code'><![CDATA[
+        Rscript
+            '$__tool_directory__/CRSconverter.R'
+            $os.path.join($input_file.extra_files_path, 'shapefile.shp')
+            $output_file_format
```

Comment:
```suggestion
            '$output_file_format'
```
I am pretty sure your are missing some single quotes here and on some other variables also but I don't want to say something wrong @bgruening  do you agree ?

Reply:
might be why I used "{}" XD